.gitignore

```diff
@@ -137,3 +137,8 @@ notebooks/
 
 # Version
 src/pseudopeople/_version.py
+
+# Make commands 
+format
```

Comment:
Is this supposed to be in .gitignore? What does it do?

Reply:
When you run make it creates a file for the command you run so we just ignore it here.